### DISCUSSION TITLE
LHR V2 Is fake? clocks and mem overclock

### DISCUSSION THREAD

**Discussion by nebutecharethebest** [NebuTech/NBMiner#494] [Ideas]
i noticed that lhr v2, i can increase my mem to +1800 and the hashrate is higher in excavator for example, and nothing bad happens

but the usage is very low, due this bios, we have a limited overclock setting
im not sure that we are using those 9000 of memory clock

maybe nvidia is fooling us

what do you think people? look this image, my hashrate is 27 with a mem overclock of 1800, wondering what happens if i set an overclock of + 4000 or 6000 overclock with a bios mod

is like if we set 1300 core clock and 12000 mem clock

the real numbers that will be used will be 1050 and 8000

dont you think people? maybe lhr v2 works in that way in ethereum excavator, didnt test this yet in nbminer

https://ibb.co/3dSw0sP

**Comment by sonicfelipemt** [comment]
I saw that my 3070ti's (hynix) mem clock won't affect real hash power when set above +1000. From +1000 to +1500, it hashes the same. 
Something strange is 100% happening. 
This might indicate that the OC is not being totally applied, there could be a limit, or the card generates fake data / alters data when delivering it to MSI Afterburner, etc. As the LHR is applied @ hardware level, who knows what's really happening.

**Comment by madsci1016** [comment]
The devs are not discussing this openly (that I've found anyway) but based on observation I believe this whole soft "unlock" of LHR is not really anything of the sort, really. They just figured out if you artificially reign in hashing you can "cheat" the LHR police a bit. Try to hash at 100% and you get caught and pushed back down to 50%. But hold yourself back at 70% and you don't get caught and LHR doesn't kick in. That is what i suspect the whole -lhr parameters do (set where you hold yourself back at) and why if you go to high you get caught and it "locks". 

So this theory explains your behavior. the miner is limiting your hashing to avoid being busted by LHR, no matter what you do with  memory clock OC (to a point).
